c7n/resources/rds.py

```diff
@@ -969,6 +969,16 @@ def augment(self, snaps):
         return snaps
 
 
+class ConfigSnapshotRDS(ConfigSource):
```

Comment:
this class is duplicate of ConfigRDS which should be used as a base class or directly instead of copy and paste.

also this impl doesn't address the requested issue